cli/src/lib/httpClient.js

```diff
@@ -0,0 +1,10 @@
+const axios = require('axios').default
+
+module.exports.createClient = ({ baseUrl, auth, ...options }) => {
+    return axios.create({
+        baseURL: baseUrl,
+        timeout: 30000,
```

Comment:
I am blissfully unaware of the context, but a 30 second timeout seems pretty extreme. Is this because it just takes a long time for an app to get"installed" onto an instance?

Reply:
I think this is as the client upload a zip-file to the instance, consider someone on a slow connection uploading a couple of megabytes?

Reply:
Yeah exactly.  I'll remove the timeout from the base client object and just specify it on the upload request.